backend/src/main/java/com/redhat/cloud/notifications/models/BehaviorGroup.java

```diff
@@ -63,6 +63,7 @@
     @JsonIgnore
     private String accountId;
 
+    @NotNull
```

Comment:
Not really in the scope of this PR, but is a minor change. This makes the `field` to be required (in the openapi)

Reply:
OpenAPI does not understand an annotation from `javax.validation.constraints`? 😢

Reply:
Might be a bug in Smallrye OpenAPI extension

Reply:
I created this issue there: https://github.com/smallrye/smallrye-open-api/issues/851 and will follow up